sktime/transformations/series/cffilter.py

```diff
@@ -0,0 +1,130 @@
+# -*- coding: utf-8 -*-
+"""
+Interface to Christiano Fitzgerald asymmetric, random walk filter from `statsmodels`.
+
+Interfaces `cf_filter` from `statsmodels.tsa.filters`.
+"""
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+
+__author__ = ["ken-maeda"]
+__all__ = ["CFFilter"]
+
+
+import pandas as pd
+
+from sktime.transformations.base import BaseTransformer
+from sktime.utils.validation._dependencies import _check_soft_dependencies
+
+_check_soft_dependencies("statsmodels", severity="warning")
+
+
+class CFFilter(BaseTransformer):
+    """Filter a times series using the Christiano Fitzgerald filter.
+
+    This is a wrapper around the `cffilter` function from `statsmodels`.
+    (see `statsmodels.tsa.filters.cf_filter.cffilter`).
+
+    Parameters
+    ----------
+    low : float
+        Minimum period of oscillations. Features below low periodicity
+        are filtered out. Default is 6 for quarterly data,
```

Comment:
the default is always 6, so this sentence is confusing. I would say "for quarterly data, the default of 6 gives 1.5 years periodicity".